projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.5-a.3 - 2021-12-14
+### Enhancements
+- JITM: allow sidebar JITMs marked as dismissible to be dismissed.
+- SSO: add filters for the error text when a local user cannot be found for a given WP.com account and for when SSO is disallowed when on a staging site.
+
+### Improved compatibility
+- Cleanup jetpack version option on plugin deactivation.
+- Notifications: remove legacy code supporting IE versions older than 9.
+- Pay with PayPal Widget: hide widget from Legacy Widget block.
+- Subscription Block: add 'Success Message Text' to the block settings.
+
+### Bug fixes
+- Admin Menu: add support for WPML menu structure.
```

Comment:
I would move this to "Other", since only Atomic site owners (who don't get to see the changelogs) are concerned by this.